examples/auction/tests/tests.rs

```diff
@@ -65,6 +65,7 @@ fn test_multiple_scenarios() {
                 message:      OwnedParameter::empty(),
             },
         )
+        .print_debug(DebugOutputKind::Full)
```

Comment:
Did you intend to include the changes in the auction example?

Reply:
Kind of, but I realize that more is needed to make this more useful in the example, so I will remove them instead.